.circleci/main.yml

```diff
@@ -118,11 +118,11 @@ jobs:
           cp go.sum go.sum.bak
 
           # make sure the examples run against the current version of go-ipfs
-          go mod edit -replace github.com/ipfs/go-ipfs=./../../..
+          go mod edit -replace github.com/ipfs/ipfs-banana=./../../..
```

Comment:
Should this just be `banana` instead of `ipfs-banana`?  In my mind prefixing with `ipfs-` implies a bit of officialness.